src/foam/nanos/notification/email/DAOResourceLoader.java

```diff
@@ -89,8 +89,10 @@ public static EmailTemplate findTemplate(X x, String name, String groupId, Strin
   }
 
   public static EmailTemplate findTemplate(X x, String name) {
-    var groupId = ((Group) x.get("group")).getId();
-    String locale = ((Subject) x.get("subject")).getRealUser().getLanguage().getCode();
+    User user = ((Subject) x.get("subject")).getRealUser();
```

Comment:
looks good. wonder how it worked before.

Reply:
why user context wouldn't have "group"

Reply:
@penzital I think emailTemplateEngine used to find  via emailTemplateDAO not DAOResourceLoader before.

Reply:
talked to @penzital and resolved.